po/it.po

```diff
@@ -1,297 +1,314 @@
+# Italian translation of pdfarranger
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the pdfarranger package.
+#
 msgid ""
 msgstr ""
-"Project-Id-Version: PDF-Shuffler 0.4\n"
+"Project-Id-Version: pdfarranger master\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-04-11 10:54+0200\n"
-"PO-Revision-Date: 2020-04-21 13:38+0100\n"
-"Last-Translator: Albano Battistella\n"
-"Language-Team: Italian\n"
-"Language: it\n"
+"POT-Creation-Date: 2020-05-24 17:01+0200\n"
+"PO-Revision-Date: 2020-07-14 08:38+0200\n"
+"Language-Team: it\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Poedit-Language: Italian\n"
-"X-Poedit-Country: ITALY\n"
+"X-Generator: Poedit 2.3.1\n"
+"Last-Translator: Michele Pezza\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: it\n"
 
-#: pdfarranger/metadata.py:35
+#: pdfarranger/metadata.py:36
 msgid "Title"
 msgstr "Titolo"
 
-#: pdfarranger/metadata.py:36
+#: pdfarranger/metadata.py:37
 msgid "Creator"
 msgstr "Creatore"
 
-#: pdfarranger/metadata.py:37
+#: pdfarranger/metadata.py:38
 msgid "Producer"
 msgstr "Produttore"
 
-#: pdfarranger/metadata.py:38
+#: pdfarranger/metadata.py:39
 msgid "Creator tool"
-msgstr "Creatore Strumento"
+msgstr "Strumento di creazione"
 
-#: pdfarranger/metadata.py:141
+#: pdfarranger/metadata.py:154
 msgid "Edit properties"
 msgstr "Modifica proprietà"
 
-#: pdfarranger/metadata.py:154
+#: pdfarranger/metadata.py:167
 msgid "Property"
 msgstr "Proprietà"
 
-#: pdfarranger/metadata.py:154
+#: pdfarranger/metadata.py:167
 msgid "Value"
 msgstr "Valore"
 
-#: pdfarranger/pdfarranger.py:520
-msgid "No document"
-msgstr "Nessun documento"
-
-#: pdfarranger/pdfarranger.py:522
-msgid "Several documents"
-msgstr "Diversi documenti"
-
-#: pdfarranger/pdfarranger.py:539
+#: pdfarranger/pdfarranger.py:585
 #, python-format
 msgid "Rendering thumbnails… [%(i1)s/%(i2)s]"
 msgstr "Disegno miniature… [%(i1)s/%(i2)s]"
 
-#: pdfarranger/pdfarranger.py:629
+#: pdfarranger/pdfarranger.py:665
+msgid "Save changes to “{}” before closing?"
+msgstr "Salvare le modifiche apportate a “{}”prima di chiudere?"
+
+#: pdfarranger/pdfarranger.py:667
+msgid "Save changes before closing?"
+msgstr "Salvare le modifiche prima di chiudere?"
+
+#: pdfarranger/pdfarranger.py:669
+msgid "Your changes will be lost if you don’t save them."
+msgstr "Le modifiche andranno perse se non le salvi."
+
+#: pdfarranger/pdfarranger.py:670
+msgid "Do_n’t Save"
+msgstr "_Non salvare"
+
+#: pdfarranger/pdfarranger.py:670
+msgid "_Cancel"
+msgstr "Annulla"
+
+#: pdfarranger/pdfarranger.py:670 data/menu.ui.h:2
+msgid "_Save"
+msgstr "Salva"
+
+#: pdfarranger/pdfarranger.py:710
 msgid "Export…"
-msgstr "Esporta…"
+msgstr "Esporta..."
```

Comment:
I think all of these ... were changed to the single character on purpose, any reason you changed it back?